### PR TITLE
set default to official ChatGPT model from OpenAI

### LINKED ISSUE #120: Will you support gpt-3.5-turbo？
When I use gpt-3.5-turbo: 
"message": "This is a chat model and not supported in the v1/completions endpoint. Did you mean to use v1/chat/completions?",

### PR DESCRIPTION
**Pull request by max298**
This is kind of a _"dangerous"_ change, as the OpenAI accounts of our users [will be charged](https://platform.openai.com/docs/guides/chat) by simply updating the version to this one.

I'm not sure if we should adjust the readme any further in order to make this clear, so everyone knows the usage is no longer free (and most likely will never become free again at this point).

Also closes #120 and #119

### CODE REVIEW

**Comment by bertybuttface** [comment]
I think we should break something deliberately before merging this. Perhaps change the namespace of the package?

**Comment by Dual-0** [comment]
maybe change the variable naming?

**Comment by bertybuttface** [comment]
> maybe change the variable naming?
> `CHATGPT_MODEL=`

Yeah that could do it. Something that requires user intervention.

**Comment by max298** [comment]
I like that idea! I'll implement the change shortly.

**Comment by max298** [comment]
I just noticed the env variable is optional and has a default, so a change of the variable will most likely not even be noticed by our users if they didn't change the model.

**Comment by bertybuttface** [comment]
Lets make it a required field and adjust the name.

**Comment by max298** [comment]
I've also added an log entry if the new `CHATGPT_API_MODEL` is unset resulting in the bot not starting.

I think this is now ready for review 😃

File: README.md [hunk 1]
```diff
@@ -7,12 +7,8 @@ Talk to ChatGPT via any Matrix client!
...
-
-- The `CHATGPT_MODEL` environment can be used to set the model - as of writing the default uses a model of ChatGPT from late 2022 which is **not working at the moment** and we do not know if and when this will change. However, you can change the `CHATGPT_MODEL` env to `text-davinci-003` or any other of the [supported models](https://platform.openai.com/docs/models/gpt-3). Keep in mind that that these alternative models are not free and will cost you OpenAI credits.
-- Check the [waylaidwanderer/node-chatgpt-api](https://github.com/waylaidwanderer/node-chatgpt-api) repository for any updates on this.
+## Warning for users upgrading from version 2.x
+OpenAI released the [official API for ChatGPT](https://openai.com/blog/introducing-chatgpt-and-whisper-apis). Thus, we no longer have to use any older models or any models which kept on being turned off by OpenAI. This means the bot is now way more stable and way faster. However, please note: The usage of the API is **no longer free**. If you use this bot, your OpenAI account **will be charged**! You might want to limit your budget in your account using the OpenAI website.
```
Comment (bertybuttface) [inline README.md:11]: should we link to https://platform.openai.com/account/billing here?

**Review by bertybuttface — APPROVED** [review]
This looks good thanks for taking the time to make this all work. Variable name looks good and sensible to me. LGTM.

File: README.md (hunk 1 above)
Comment (Dual-0) [reply README.md:11]: Can we catch the warning, the API returns when the limit is reached? Would be helpful if the bot returns a message like "API usage reached..."

Comment (Dual-0) [reply README.md]: btw: you missed line 8.  `to access the unofficial ChatGPT API.`. Should be the `official API`.

**Pull request merged**